include/gatekeeper_cps.h

```diff
@@ -126,20 +129,37 @@ struct cps_request {
 
 struct route_update {
 	/* Type of update: RTM_NEWROUTE or RTM_DELROUTE. */
-	int type;
+	int      type;
 
 	/* Address family of update: AF_INET or AF_INET6. */
-	int family;
+	int      family;
 
 	/*
 	 * Whether this update has all the fields and attributes
 	 * necessary to update the LPM table.
 	 */
-	int valid;
+	int      valid;
+
+	uint8_t  prefix_len;
+
+	uint32_t oif_index;
+
+	union {
+		struct in_addr  v4;
+		struct in6_addr v6;
+	} ip;
+
+	union {
+		struct in_addr  v4;
+		struct in6_addr v6;
+	} gw;
 };
 
+/* TODO Patch BGP speakers to redirect the editing commands to Gatekeeper. */
+
```

Comment:
We don't need this `TODO` since we already have issue #56 for that. More importantly, given that patching BGP speakers won't/shouldn't require a patch for Gatekeeper, this `TODO` wouldn't be obviously removed.